pyproject.toml

```diff
@@ -3,7 +3,7 @@ name = "armada_jupyter"
 version = "0.1.0"
 description = "Submit JupyterLab Pods to Armada"
 readme = "README.md"
-requires-python = ">=3.7,<3.10"
+requires-python = ">=3.7,<3.9"
```

Comment:
I'm not up to speed here, why are newer versions of Python not supported?

Reply:
See the above :)